test/controllers/flag_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class FlagControllerTest < ActionController::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
We should add some basic test coverage here.  I have used the `shoulda` gem which has some nice helpers for testing relationships- see the other models.